.github/workflows/manylinux/test_package_i686.sh

```diff
@@ -36,17 +34,13 @@ $PYTEST_COMMAND
 # onnx.checker all existing backend data
 $PYTHON_COMAND workflow_scripts/test_generated_backend.py
 # onnx.checker all generated backend data
-$PYTHON_COMAND onnx/backend/test/cmd_tools.py generate-data
-$PYTHON_COMAND workflow_scripts/test_generated_backend.py
-
-# Verify ONNX with the latest numpy
-if [ "$PY_VERSION" == "3.8" ] || [ "$PY_VERSION" == "3.9" ]; then
-    $PIP_UNINSTALL_COMMAND numpy onnx && $PIP_INSTALL_COMMAND numpy==1.21.5
-else
-    $PIP_UNINSTALL_COMMAND numpy onnx && $PIP_INSTALL_COMMAND numpy
+# FIXME: Python 3.10 wheel runs forever during generate-data; skip for now
```

Comment:
https://github.com/numpy/numpy/issues/20277 NumPy didn't provide prebuilt Python 3.10 i686 wheel. Since now ONNX uses NumPy 1.21.5 in general, it needs to build NumPy from source for Python 3.10, which is probably not a good idea. Besides, this built Python 3.10 i686 ONNX wheel seems problematic: it hangs a long time while generating data. It took me quite a while to investigate what the root cause is, but I still don't know why.

And in newer NumPy (>1.22), they even don't provide i686 wheel for all Python versions anymore. I think we should reconsider to deprecate manylinux i686 packages since people are retiring them and the release CI for i686 will be unstable. If we decide to do so, it needs to be done early before next release to let users be aware.

Reply:
Created an issue https://github.com/onnx/onnx/issues/4049 for tracking. I tend to skip Python 3.10+i686 for now in this PR and then we can at least let CIs cover Python 3.10 in other environments.